devicon.json

```
@@ -355,7 +357,7 @@
                 "line-wordmark"
             ]
         },
-        "color": "#303284",
+        "color": "#aa0001",
```

Is this the right color?
Seems to me like `#BD202E` is the most dominant color in the wordmark icon

---

It is from text, not icon. Icon has bunch of gradients.

---

I couldn't find `#aa0001` anywhere in the icon's code. Is it the average of the gradient or something?

---

Double checked it. Actually, logo use neither of those colors, main color for text varies, depending on which logo is used as base.

In icon used in original pull request it is actually `#aa0000`: https://svn.apache.org/repos/asf/comdev/project-logos/originals/httpd.svg

However, due to issues with "A" letter, I've rebuilt icon using this as source: https://svn.apache.org/repos/asf/comdev/project-logos/originals/foundation.svg and it uses `#cb2533` for text.

So correct color is `#aa0000` or `#cb2533`, depending on which Apache project it refers to.

---

Ahh, right.
I suggest using `#cb2533` then

```suggestion
        "color": "`#cb2533`",
```

---

Sure... but it went a bit wrong. Noticed suggestion, clicked button to commit it, only later noticed that color in suggestion included quotes.

---

Fixed that in a separate commit, force-pushed to get rid of bad commit.

---

The issue happens when I said [here](https://github.com/devicons/devicon/pull/1527#issuecomment-1321879664) about APACHE HTTP SERVER and the text of this one is filled with `#aa0000`, which is practically the same as `#aa0001`, but APACHE SOFTWARE FOUNDATION, which I based my fix, have `#BD202E` as the major color.
So I'll be re-suggesting using `#BD202E`

---

"Apache Software Foundation" logo uses `#cb2533` for text.

Color in icon is irrelevant because there is no fixed color. It is all gradients.